openapi-cli/src/test/resources/ballerina-to-openapi/data_type/nullable_type_def.bal

```diff
@@ -0,0 +1,22 @@
+import ballerina/http;
+
+public type Action string?;
+public type Count decimal?;
+public type Rels string[]?;
+public type Books map<string>?;
```

Comment:
Shall we add test cases and cover int, float and union types related scenarios as well?

Reply:
When covering union types, lets add a type that contains more than 2 types (to validate the recursive logic in your implementation)

Reply:
Yeah,  The second test has covered with `int` type and will add float and more union type to test :)